agent/util-scripts/pbench-register-tool-set

```diff
@@ -129,6 +129,63 @@ if [[ -z ${the_tool_set} ]]; then
 	exit 1
 fi
 
+# this code is a cut-and-paste from pbench-register-tool
+# If we are given a remotes file, check it for sanity now,
+# before trying to call pbench-register-tool and failing (multiple times).
+if [[ ${remotes_arg::1} == "@" ]]; then
+	# We are dealing with a file containing the list of remotes.
+
+	if [[ -n ${labels_arg} ]]; then
+		error_log "--labels=${labels_arg} not allowed with remotes file (${remotes_arg})"
+		usage >&2
+		exit 1
+	fi
+	remotes_file=${remotes_arg#*@}
+	if [[ ! -e ${remotes_file} ]]; then
```

Comment:
Maybe `-r`.

 AFAICT, an empty file is allowed: no tools are registered (but the `${pbench_run}/tools-v1-<toolgroup>` directory is created if necessary) and we get a normal exit, from both `pbench-register-tool-set` and `pbench-register-tool`. I think I'll open an issue for this one: it's probably an error, but it will need more unrelated changes than I want to make in this PR.

Reply:
Opened issue #2538 to address the empty remotes file case.